app/helpers/application_helper/button/custom_button.rb

```diff
@@ -0,0 +1,8 @@
+class ApplicationHelper::Button::CustomButton < ApplicationHelper::Button::Basic
```

Comment:
This class is not needed. The only method redefined has the same content in the base class.

Why is it defined? and why does it include the ApplicationHelper?